### PR TITLE
[APP 707] general setting components

### PR DESCRIPTION
**Pull request by nirbhayel**

### CODE REVIEW

File: modules/settings/assets/js/components/color-picker/style.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.custom-color-picker.react-colorful {
```
Comment (bainternet) [inline modules/settings/assets/js/components/color-picker/style.css:1]:
```suggestion
.widget-settings-color-picker.react-colorful {
```

let give it a more specific class then just "custom-color-picker"

**Review by bainternet — CHANGES_REQUESTED** [review]
please see comment above and are we sure that this is RTL ready??

**Review by bainternet — APPROVED** [review]

File: modules/settings/assets/js/components/color-picker/style.css
Comment (bainternet) [inline modules/settings/assets/js/components/color-picker/style.css:1]: you only changed the first one!

what about the rest of `.custom-color-picker`?

**Review by bainternet — CHANGES_REQUESTED** [review]
please update styles

File: modules/settings/assets/js/components/color-picker/style.css
Comment (nirbhayel) [reply modules/settings/assets/js/components/color-picker/style.css:1]: updated

**Pull request merged**